2_ThreeSum.cpp

```diff
@@ -62,31 +62,31 @@ using namespace std;
 vector<vector<int> > threeSum(vector<int> &num)
 {
 	//sort the array
-	sort(num.begin(), num.end());
+    sort(num.begin(), num.end());
     //ret contains the resultant triplets
     vector<vector<int> > res;
     //tuple format of the result
     vector<int> tuples(3);

```

Comment:
You always want to declare variables in as small of a scope as possible to make sure they don't accidentally get used before they are initialized or after their usefulness has expired.

Reply:
Will keep this in mind. Thankyou.